### PR TITLE
Add TriBITS paragraph to framework section

### PR DESCRIPTION
**Pull request by bartlettroscoe**
CC: @searhein, @srajama1

This is the same branch as for the PR:

*  https://github.com/jwillenbring/trilinos-paper/pull/1

which builds off of the branch in the PR:

* https://github.com/searhein/trilinos-paper/pull/27

to avoid a merge conflict with @jwillenbring's branch.

It would be better to first merge the PR https://github.com/jwillenbring/trilinos-paper/pull/1 and then merge the PR https://github.com/searhein/trilinos-paper/pull/27.  But I added this PR to make sure these commits don't get lost.

### CODE REVIEW

File: framework.tex [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+This allows a large number of (Trilinos) CMake packages to be configured, built, and tested in a flexible and efficient manner.
+In addition, TriBITS provides support for a number of advanced features that are not available in raw CMake/CTest including: eliminating a large amount of boiler-plate CMake code and avoiding common mistakes; enabling and testing all downstream packages given a set of enabled (i.e. modified) upstream packages; managing the enable and disable of tests based on various criteria; producing build and test results submitted to a CDash site on a package-by-package basis; producing reduced source tarballs for only a desired subset of enabled packages.
+As of Trilinos 14.4, TriBITS and Trilinos have been updated to allow integrating packages using raw CMake with just a few well-defined integration requirements.
+The TriBITS framework has allowed Trilinos to scalably grow in the number of packages and the complexity without undo burdening of individual Trilinos developers and users.
```
Comment (mayrmt) [inline framework.tex:18]: Typo?
```suggestion
The TriBITS framework has allowed Trilinos to scalably grow in the number of packages and the complexity without undue burdening of individual Trilinos developers and users.
```

File: main.tex [hunk 2]
```diff
@@ -154,6 +154,8 @@
 \email{hkthorn@sandia.gov}
 \author{James M. Willenbring}
 \email{jmwille@sandia.gov}
+\author{Roscoe A. Bartlett}
+\email{rabartl@sandia.gov}
 \orcid{0000-0002-0418-9264}
```
Comment (mayrmt) [inline main.tex:159]: @bartlettroscoe I believe that the ordering needs to be as follows, such that the ORCID of @jwillenbring will be associated with his name:
```suggestion
\author{James M. Willenbring}
\email{jmwille@sandia.gov}
\orcid{0000-0002-0418-9264}
\author{Roscoe A. Bartlett}
\email{rabartl@sandia.gov}
```

Comment (bartlettroscoe) [reply main.tex]: @mayrmt, thanks for catching that!  I fixed Jim's ORCID and I added mine in the new commit https://github.com/searhein/trilinos-paper/pull/28/commits/e6ed53d4e75a09b3b225f6dcf26afeaf12f0b299

**Review by mayrmt — APPROVED** [review]

**Review by mayrmt — APPROVED** [review]
I think, this can be merged.

**Pull request merged**